services/webdav/pkg/config/config.go

```diff
@@ -18,9 +18,8 @@ type Config struct {
 
 	HTTP HTTP `yaml:"http"`
 
-	OcisPublicURL   string `yaml:"ocis_public_url" env:"OCIS_URL;OCIS_PUBLIC_URL" desc:"URL, where oCIS is reachable for users."`
-	WebdavNamespace string `yaml:"webdav_namespace" env:"WEBDAV_WEBDAV_NAMESPACE" desc:"CS3 path layout to use when forwarding /webdav requests"`
-	RevaGateway     string `yaml:"reva_gateway" env:"REVA_GATEWAY" desc:"The CS3 gateway endpoint."`
-
-	Context context.Context `yaml:"-"`
+	OcisPublicURL   string          `yaml:"ocis_public_url" env:"OCIS_URL;OCIS_PUBLIC_URL" desc:"URL, where oCIS is reachable for users."`
+	WebdavNamespace string          `yaml:"webdav_namespace" env:"WEBDAV_WEBDAV_NAMESPACE" desc:"CS3 path layout to use when forwarding /webdav requests"`
+	Reva            shared.Reva     `yaml:"reva"`
```

Comment:
Similar to the notifications service this is also backwards incompatible.

Reply:
`OCIS_URL;OCIS_PUBLIC_URL`
just reminds me that we still do not have the mechanism to publish envs starting with `OCIS_` automatically - means that we have gaps in the documentation how to deal with it @dragonchaser fyi

Reply:
@mmattel If you don't want this to get lost, I suggest to open a separate issue for this.